Lab3/Source.cpp

```diff
@@ -0,0 +1,71 @@
+
+#include <windows.h>
+#include <iostream>
+using namespace std;
+int _size;
+int max = INT_MIN, min = INT_MAX;
+int averageElement = 0;
```

Comment:
do not use global variables as return values, introduce some params struct instead